dashboard/app/views/plc/enrollment_task_assignments/complete/_learning_resource_task.html.haml

```diff
@@ -0,0 +1,4 @@
+%p Hello, for this task, just check out this website
+
+%p
+  %iframe{width: '800px', height: '800px', src: 'http://' + @task.resource_url}
```

Comment:
Okay - I'm not too worried about this part, it's mostly there to prove that the links are actually there. We don't actually have a defined UX for what this should look like.

For the sake of the security restriction, I'll drop the iframe and replace it with just a link.